test.js

```diff
@@ -6,4 +6,10 @@ test(t => {
 	t.is(m(65, 131, 196), '4183c4');
 	t.is(m('255 154 253'), 'ff9afd');
 	t.is(m('rgb(40, 42, 54)'), '282a36');
+	/* eslint no-floating-decimal:0 */
+	t.is(m(0, 0, 0, .5), '00000080');
```

Comment:
`.5` => `0.5`. There's no point in testing without a `0` as the module doesn't see anything different here.